bullet/client.py

```diff
@@ -131,16 +131,24 @@ def __init__(
             raise ValueError("Indent must be > 0!")
         if margin < 0:
             raise ValueError("Margin must be > 0!")
-
+        
         self.prompt = prompt
-        self.choices = choices
+        
+        #test flag also return objects
+        self.objects=[]
+        if(type(choices) is dict):
+            self.choices = list(choices.keys())
+            self.objects=[choices[x] for x in self.choices]
```

Comment:
How about `list(choices.values())` for consistency?

Reply:
did not know that there was a .values() function